DXRModules/DeusEx/Classes/DXREventsBase.uc

```diff
@@ -687,14 +688,29 @@ function _AddPawnDeath(ScriptedPawn victim, optional Actor Killer, optional coer
         }
     }
 
+    name = string(victim.alliance);
+    _MarkBingo(name$"_AllianceEliminated");
```

Comment:
This should probably differentiate between dead and unconscious also, so _AllianceDead and _AllianceUnconscious